styles/css/core/variables.css

```diff
@@ -30,27 +30,27 @@
   --pgn-elevation-modal-zindex: 1050;
   --pgn-elevation-modal-backdrop-zindex: 1040;
   --pgn-elevation-dropdown-zindex: 1000;
-  --pgn-transition-collapse-width-behavior: unset; /* Collapse transition for width that takes 350ms */
+  --pgn-transition-collapse-width-behavior: normal; /* Collapse transition for width that takes 350ms */
```

Comment:
[inform]: When rechecking the documentation site I noticed that some transitions are not working. CSS transition is incorrect due to the use of `unset` value, replaced `unset` with `normal `

Reply:
Looks good. I am curious about which transitions you found that weren't working before the change. I'm wondering if the shorthand we were using before led to some having `transition-behavior` set to `allow-discrete` instead of `normal` ([MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/transition-behavior))

Reply:
While checking the Form.Control component, I noticed issues with transitions. It seemed like the transition was missing, so I decided to copy the value of the CSS variable var(--pgn-transition-form-input) and manually apply it to the transition property. However, this resulted in an incorrect value, indicating that the var(--pgn-transition-form-input) variable holds the wrong value.

![image](https://github.com/user-attachments/assets/e2545d82-8a07-428a-b3ac-dc7298666e5d)

After this experiment, I realized that the value `unset` is not valid for transitions. I have replaced it in all tokens related to transitions.

Reply:
Thanks for the added context! I checked on the docs site to see the pre-design-tokens transition value for `Form.Control` and it does appear to be using `transition-behavior: normal;`.

![image](https://github.com/user-attachments/assets/97366b58-5a7b-4622-9b1d-f1bac6308fce)

 I don't know of any places we use `transition-behavior: allow-discrete` so I think explicitly setting it to `normal` here should be fine.